qa/tasks/ceph_objectstore_tool.py

```diff
@@ -514,7 +514,7 @@ def test_objectstore(ctx, config, cli_remote, REP_POOL, REP_NAME, ec=False):
                 proc.wait()
                 if proc.exitstatus != 0:
                     log.error("Failure of --op info command with {ret}".
-                              format(proc.exitstatus))
+                              format(ret=proc.exitstatus))
```

Comment:
could you split changes like this into another commit? also, i'd suggest use the f-string, like:
```py
log.error(f"Failure of --op info command with {proc.exitstatus}")
```

Reply:
Why splitting? It is fixing problems detected by mypy.

Reply:
fstrings are python3 only but we currently still have python2